L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigisPostLS1_cfi.py

```diff
@@ -1,5 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
+print " ######################################################################################################### "
```

Comment:
Done. In a follow-up PR I'm planning  to delete this file and clean-up cscTriggerPrimitiveDigis_cfi to suit the era customizations.